octue/cli.py

```diff
@@ -7,12 +7,16 @@
 from octue.runner import Runner
 
 
+context = {}
```

Comment:
Seems to work, but I'd anticipated using the [context object provided by clicks](https://click.palletsprojects.com/en/7.x/complex/#contexts).

Do you think that'd give us any benefits (e.g. if calling the CLI from python we'd be reusing the object, but since the key values are overwritten then copied at each invocation, I don't think this'd lead to any issues??).

Reply:
We can definitely use the Click context objects if you prefer, although it seemed a bit complicated (the main reason I went for the global dict is that I couldn't initially see how to use the Click context).

I think the benefit of the dict is simplicity. I don't at this point see a way it could be abused or misused by a user (unless it's by you or me!), and it removed the need for multiple decorators and extra parameters.

Reply:
Agreed, don't see the need for extra complexity where there is none.

And also agreed that users won't be able to abuse it in any way (accidentally or intentionally).

I suppose we could just give our poor tired future selves a hint? Like call it (say) global_cli_context or something?